_sass/uno.scss

```diff
@@ -1108,10 +1107,12 @@ pre {
 
 .footer {
 	display: block;
-	padding: 2em 0 0 0;
-	border-top: 2px solid #dddddd;
+	padding: 17px 0 6px 0;
+	background: #f9f9f9;
 	font-size: .7em;
-	color: #b3b3b3; }
+	margin-left: 530px;
```

Comment:
Does this need to be fixed? Should this vary for screen size?

Reply:
This footer was giving me a lot of grief. I've reached a compromise.